onnxscript/function_libs/torch_aten/ops/special.py

```diff
@@ -202,12 +204,13 @@ def aten_special_log_ndtr(self: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
+@torch_op("aten::log_softmax")
 def aten_special_log_softmax(
-    self: TensorType, dim: int, dtype: Optional[int] = None
+    self: TensorType, dim: int, dtype: int = -1  # pylint: disable=unused-argument
```

Comment:
```suggestion
    self: TensorType, dim: int, dtype: int = -1
```

possible to use dtype in the logic?